extras/basic_example/public/script.js

```diff
@@ -5,17 +5,39 @@
 /* eslint-disable no-param-reassign, no-console */
 
 const serverUrl = '/';
-window.localStream = undefined;
-window.room = undefined;
-window.recording = false;
-window.recordingId = '';
+let localStream;
+let room;
+let recording = false;
```

Comment:
We are not setting `recording` and `recordingId` to `window` anymore. I think it is good because we don't have a case that needs them as window parameters.

Reply:
👍  That was my reasoning but I'll add them for consistency